.github/workflows/cd-dev.yml

```
@@ -0,0 +1,39 @@
+name: Yes dev CD 파이프라인
+
+on:
+  workflow_run:
+    workflows: ["Yes CI 파이프라인"]
+    types:
+      - completed
+
+jobs:
+  build:
```

```suggestion
  deploy:
```
다음과 같이 job 이름을 변경해 역할을 보다 명확히 하면 좋을 것 같습니다.